README.md

```diff
@@ -18,8 +18,6 @@ The class’ constructor takes one required parameter (a configured imp I&sup2;C
 | *i2cAddress*  | byte         | 0x30    | The I&sup2;C address of the accelerometer |
 
 ```squirrel
-#require "LIS3DH.class.nut:1.3.0"
-
```

Comment:
As mentioned above, I believe `init`should be renamed to `reset` or something. And in the docs we should be clear with when to call it and why. I.e. you don't have to call it before the library initialization. It's in fact a reset functionality.